keras_hub/src/models/vit/vit_backbone.py

```diff
@@ -74,24 +80,31 @@ def __init__(
                 f"at index {h_axis} (height) or {w_axis} (width). "
                 f"Image shape: {image_shape}"
             )
-        if image_shape[h_axis] != image_shape[w_axis]:
+
```

Comment:
could consider

```python
if isinstance(patch_size, int):
   patch_size = (patch_size, patch_size)
```

for compat. it's not too burdensome

Reply:
Yes, this is should be good.